src/backends/beamcoder.ts

```diff
@@ -338,6 +338,7 @@ export class BeamcoderExtractor extends BaseExtractor implements Extractor {
                 if ((nextFrame && nextFrame.pts > targetPTS) || (closestFramePTS === targetPTS)) {
                     // We have a next frame, so we know the frame being displayed at targetPTS is the previous one,
                     // which corresponds to outputFrame.
+                    this.#previousTargetPTS = targetPTS;
```

Comment:
We must update `previousTargetPTS` each time we update or return `outputFrame`. Otherwise, the system can't know we're looking backwards and thus doesn't seek!